types/k6/experimental/fs.d.ts

```diff
@@ -0,0 +1,102 @@
+/**
+ * The k6 File System (fs) module provides an API for handling file operations 
```

Comment:
```suggestion
 * k6 experimental filesystem module provides an API for handling file operations 
```
s/File System/filesystem/

for being consistent with your writes on the documentation